components/Collections.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  Button,
+  ButtonGroup,
+  Spinner,
+  Stack,
+  Text,
+  VStack,
+} from '@chakra-ui/react'
+import { getServicesList } from 'api'
+import { ServicesList } from 'models'
+import { UserDoc } from 'models/users'
+import React, { useEffect, useState } from 'react'
+import CollectionItem from './CollectionItem'
+
+interface CollectionProps {
+  userData: UserDoc
+}
+
+export const Collections = ({ userData }: CollectionProps): JSX.Element => {
+  const [loading, setLoading] = useState(true)
+  const [listData, setListData] = useState<ServicesList[]>([])
+
+  useEffect(() => {
+    const promises = []
+    for (const list of userData.lists) {
+      const listData = getServicesList(list)
+      promises.push(listData)
+    }
+
+    Promise.all(promises.map((p) => p.catch((e) => e))).then((list) => {
+      setListData(list.filter((l) => !(l instanceof Error)))
+      setLoading(false)
+    })
+  }, [userData.lists])
+
+  if (loading) {
+    return (
+      <Stack w="100%" alignItems={'center'}>
+        <Spinner variant="primary" />
+      </Stack>
+    )
+  }
+
+  return (
+    <VStack alignItems="left" w="100%" spacing={4}>
+      <ButtonGroup variant={'ghost'}>
+        <Button as="a" href="/">
+          Create New Collection
+        </Button>
+        <Button as="a" href="/">
+          Edit Collections
+        </Button>{' '}
+      </ButtonGroup>
+
+      {listData.map((list) => (
+        <CollectionItem key={list.id} servicesList={list} />
+      ))}
```

Comment:
Should we be displaying this in a grid similar to the homepage? Maybe a q for design (if we still have a designer 😢)

Reply:
I did it without using the grid because it was easier to make it responsive to mobile but that's a good point. Also yeah we could consolidate it if you want but I think the way it is right now is fine

Reply:
Up to you if you wanna consolidate in this PR or later but we can make our grids responsive so the look good on mobile. Not sure if it's better to do the mobile designs as we go or pray that we have mobile designs and do it all later.